test/test_trixi2vtk.jl

```diff
@@ -10,25 +11,153 @@ function run_trixi(elixir; kwargs...)
 end
 
 
-function sha1file(filename)
-  open(filename) do f
-    bytes2hex(sha1(f))
+"""
+    get_reference_file(filename, remotename; head="main", output_directory=".", force=false)
+
+Retrieve an reference file from the
+[`Trixi2Vtk_reference_files` repository](https://github.com/trixi-framework/Trixi2Vtk_reference_files)
+at commit/branch `head` and store it in the `output_directory`. If the file already
+exists locally, do not download the file again unless `force` is true.
+Provide the `remotename` because the file structure in the `Trixi2Vtk_reference_files` repository
+is different than what will be created locally. Return the local path to the downloaded file.
+"""
+function get_reference_file(filename, remotename; head="main", output_directory=".", force=false)
+  filepath = joinpath(output_directory, filename)
+  if !isfile(filepath) || force
+    url = ("https://github.com/trixi-framework/Trixi2Vtk_reference_files/raw/"
+           * head
+           * "/reference_files/"
+           * remotename)
+    download(url, filepath)
   end
+
+  return filepath
 end
 
+# Commit in the reference file repository for which the test files will be downloaded
+# Note: The purpose of using a specific commit hash (instead of `main`) is to be able to tie a given
+#       version of Trixi2Vtk to a specific version of the test file repository. This way, also tests
+#       for older Trixi2Vtk releases should continue to work.
+TEST_REFERENCE_COMMIT = "c0a966b06489f9b2ee3aefeb0a5c0dae733df36f"
+
+# Local folder to store downloaded reference files. If you change this, also adapt `../.gitignore`!
+TEST_REFERENCE_DIR = "reference_files"
+
+
+get_test_reference_file(filename, remotename) = get_reference_file(filename, remotename,
+                                                                   head=TEST_REFERENCE_COMMIT,
+                                                                   output_directory=TEST_REFERENCE_DIR)
+
+
+# Start with a clean environment: remove reference file directory if it exists
+isdir(TEST_REFERENCE_DIR) && rm(TEST_REFERENCE_DIR, recursive=true)
+mkpath(TEST_REFERENCE_DIR)
+
+
+"""
+     compare_cell_info(out_filename, ref_filename; atol=500*eps(), rtol=sqrt(eps()))
+
+Test values from the VTK file header and acutal (possibly interpolated) cell data. Uses
+`out_filename` created during testing and compares against `ref_filename` that comes
+from the
+[`Trixi2Vtk_reference_files` repository](https://github.com/trixi-framework/Trixi2Vtk_reference_files).
+"""
+function compare_cell_info(out_filename, ref_filename; atol=500*eps(), rtol=sqrt(eps()))
```

Comment:
When I read "compare cell info", in my mind I think that this is just about information on the cells themselves (cell types etc.). We do this too, but the main motivation is really to compare the cell-based *data*, which is also the VTK name. I'd thus prefer to rename this, but I have no strong feelings...
```suggestion
"""
     compare_cell_data(out_filename, ref_filename; atol=500*eps(), rtol=sqrt(eps()))

Test values from the VTK file header and acutal (possibly interpolated) cell data. Uses
`out_filename` created during testing and compares against `ref_filename` that comes
from the
[`Trixi2Vtk_reference_files` repository](https://github.com/trixi-framework/Trixi2Vtk_reference_files).
"""
function compare_cell_data(out_filename, ref_filename; atol=500*eps(), rtol=sqrt(eps()))
```

Reply:
Okay, I have no opinion on this. We can change it but I will do it in its own commit because it touches a lot of code lines

Reply:
Do we also want to change the other function name to `compare_point_data`?

Reply:
Yes, I would change both.